Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
The reasons for bundling lua/sync internally with rpm (statically linked), has diminished over the years. If there is no need for it (local patches), then might as well use the regular formula/shared.